File: spec/code_samples/Node native/exchange-keys@{keyId}@activate/delete.js
var http = require("http");

var options = {
  "method": "DELETE",
  "hostname": "ec2-3-222-208-106.compute-1.amazonaws.com",
  "port": "8086",
  "path": "/exchange-keys/%7BkeyId%7D/activate",
  "headers": {}
};

var req = http.request(options, function (res) {
  var chunks = [];

  res.on("data", function (chunk) {
    chunks.push(chunk);
  });

  res.on("end", function () {
    var body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: spec/code_samples/Python python3/exchange-keys@{keyId}/delete.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("DELETE", "/exchange-keys/%7BkeyId%7D")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))